WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -611,6 +622,17 @@ private extension OrderDetailsViewController {
         }
     }
 
+    private func connectToCardReader() {
+        guard let viewController = UIStoryboard.dashboard.instantiateViewController(ofClass: CardReaderSettingsPresentingViewController.self) else {
+            fatalError("Cannot instantiate `CardReaderSettingsPresentingViewController` from Dashboard storyboard")
+        }
+
+        let viewModelsAndViews = CardReaderSettingsViewModelsOrderedList()
```

Comment:
This is an interesting spot where you might decide to have a different set of viewmodels, since you don't need to show the connected view for example. Don't need to do this now, just a thought for the future.

Reply:
Ah, that's a good point, I love this.